<launch>
  <node name="plc_constructor_node" pkg="plc_message_constructor" type="constructor_node" output="screen">
    <rosparam file="$(find plc_message_constructor)/config/plc_constructor_config.yaml" command="load" />
  </node>
</launch>

